src/ActiveCollab/EmailReplyExtractor.php

```diff
@@ -32,7 +33,7 @@ public static function extractReplyEml($path)
 
       $extractor = self::getExtractorEml(self::detectMailer(self::getHeadersRelevantForMailerDetectionEml($parser)), $parser);
 
-      return (string) $extractor->body;
+      return (string) trim($extractor->body);
```

Comment:
`trim()` always returns string. If you are going to cast the value, cast it before it enters `trim()`.